Resources/views/Form/form_ckeditor_layout.html.twig

```diff
@@ -20,4 +20,28 @@
     CKEDITOR.replace('{{ form.vars.id }}', options);
 </script>
 {% endspaceless %}
-{% endblock %}
\ No newline at end of file
+{% endblock %}
+
+{% block fsi_map_widget %}
```

Comment:
I'd move this block to separate template and add separate compiler pass to register it as a form resource